docs/internals/releases.md

```diff
@@ -40,13 +40,34 @@ synchronization:
   is pushed to <https://github.com/tigerbeetle/tigerbeetle-go>
 - if publishing to all registries were successfully, the release is marked as non-draft.
 
-All publishing keys are stored as GitHub Actions secrets.
+All publishing keys are stored as GitHub Actions in the `release` environment.
 
 ## Triggering a Release
 
-Releases are triggered manually, on Monday, by pushing the the current tip of `origin/main` to
+Releases are triggered manually, on Monday, by pushing a commit from `origin/main` to
```

Comment:
Maybe include an example command? I assume it is something like `git push origin origin/main:origin/release`.

Reply:
Good point! The overall plan is to polish the exact sequence of command manually, and then move the thing to a zig script as well, as I did for rust-analyzer a while back: 

https://github.com/rust-lang/rust-analyzer/blob/0890512e1c898c9c4c271df12d43353f3b64daf3/xtask/src/release.rs#L12-L15